conf/modules.config

```diff
@@ -39,19 +39,22 @@ process {
 
     withName: TRIMGALORE {
         ext.args = [
-            '--fastqc',
+            /*'--fastqc', gonna keep fastqc in its own module for reasons*/
+            '--illumina',
+            params.trim_quality > 0 ? "--quality ${params.trim_quality}" : '',
```

Comment:
I like the way you generalized this. Will keep this in mind for the future. Yay learning!